### PR TITLE
M3-4830: Breadcrumb styles

### PR DESCRIPTION
**Pull request by tiffwong**
## Description

Decrease font size of breadcrumbs and change color

**Update:** 

Fixed the styling for mobile Linodes and Kubernetes Detail breadcrumbs and added edit functionality to Stackscripts and Domains Detail breadcrumbs with mobile styling

The Domains Detail breadcrumb takes a second to actually update the label, not sure if there's a way to speed this up 🤔 

## Note to Reviewers

Please check for different viewports and error messages in the editable breadcrumb

### CODE REVIEW

**Review by Jskobos — CHANGES_REQUESTED** [review]
Looks much nicer this way. In some places--I noticed on LKE detail--the error state has disappeared in this PR (try to rename your cluster to a blank string). On Linodes detail, the error state is present but looks funky (technically not new to this PR but we should fix it):

Screen Shot 2021-01-22 at 8 48 13 AM

**Comment by ndukatz** [comment]
the edit looks way better. I did notice that if the text is long, it starts you at the end instead of the beginning of the text. This is most likely working as designed but I just wanted to make sure 
Screen Shot 2021-01-25 at 1 41 29 PM
.

**Comment by ndukatz** [comment]
> Looks much nicer this way. In some places--I noticed on LKE detail--the error state has disappeared in this PR (try to rename your cluster to a blank string). On Linodes detail, the error state is present but looks funky (technically not new to this PR but we should fix it):
> 
> Screen Shot 2021-01-22 at 8 48 13 AM

I am still seeing this same behavior on kubernetes page. the error text flashes for a second then disappears. too fast for a user to see.

**Review by DevDW — COMMENTED** [review]
Looking good overall so far -- should the StackScript and Domain labels be editable inline as it is for Linode labels and the others?

Also, the error needs some responsive styling for smaller viewports:

Screen Shot 2021-01-25 at 5 24 56 PM

**Review by Jskobos — APPROVED** [review]
I agree with the mobile styling fixes and making Domains etc. editable (also OBJ buckets). Otherwise looks good to go, going to approve pending those changes.

**Review by ndukatz — APPROVED** [review]
Looks really good to me now. great job!

**Comment by tiffwong** [comment]
@ndukatz @DevDW @Jskobos thanks for the feedback!

If the label is long, I think starting at the end is fine and there is currently no way to update a bucket name

File: packages/manager/src/features/Domains/DomainDetail/DomainDetail.tsx [hunk 1]
```diff
@@ -53,11 +57,29 @@ const DomainDetail: React.FC<CombinedProps> = props => {
...
   React.useEffect(() => {
     refreshDomainRecords();
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const handleLabelChange = (label: string) => {
+    setUpdateError(undefined);
+
+    return domainActions
+      .updateDomain({ domainId: domain.id, domain: label })
```
Comment (Jskobos) [inline packages/manager/src/features/Domains/DomainDetail/DomainDetail.tsx:71]: updateDomain is already included in this component via useDomains (line 55). No need to use the wrapper you've added.

**Review by Jskobos — COMMENTED** [review]
Everything is working, I left one comment that should be addressed but otherwise good to go.

**Pull request merged**